### PR TITLE
[WIP] Remove ObjectController references [v1.11]

### PR DESCRIPTION
**Pull request by Serabe**
Same as #90, but for v1.11 branch.

### CODE REVIEW

File: source/concepts/naming-conventions.md [hunk 1]
```diff
@@ -189,7 +189,9 @@ template with the app's instance of `app/controllers/post.js`.
 If you don't specify the controller (`app/controllers/post.js`),
 Ember will automatically make one for you based on the return value
 of the route's `model` hook. If the model is an Array, you get an
-`ArrayController`. Otherwise, you get an `ObjectController`.
+`ArrayController`. Otherwise, you get an `ObjectController`. As of 1.11,
+`ObjectController`s are deprecated and if you try to use its proxy feature you
+will get a warning.
```
Comment (locks) [inline source/concepts/naming-conventions.md]: I would link to the deprecation listing here.

**Comment by trek** [comment]
@Serabe looks like this needs a rebase.

**Comment by Serabe** [comment]
Done.

File: source/controllers/index.md [hunk 2]
```diff
@@ -110,15 +110,10 @@ makes it easy to separate display-specific properties from model
...
-For convenience, Ember.js provides controllers that _proxy_
-properties from their models so that you can say `{{name}}` in your
-template rather than `{{model.name}}`. An `Ember.ArrayController`
-proxies properties from an Array, and an `Ember.ObjectController`
-proxies properties from an object.
-
-If your controller is an `ArrayController`, you can iterate directly
-over the controller using `{{#each item in controller}}`. This keeps the
-template from having to know about how the controller is implemented
+For convenience, Ember.js provides and `Ember.ArrayController` that proxies
```
Comment (trek) [inline source/controllers/index.md]: s/and/an

File: source/concepts/naming-conventions.md [hunk 3]
```diff
@@ -189,7 +189,10 @@ template with the app's instance of `app/controllers/post.js`.
 If you don't specify the controller (`app/controllers/post.js`),
 Ember will automatically make one for you based on the return value
 of the route's `model` hook. If the model is an Array, you get an
-`ArrayController`. Otherwise, you get an `ObjectController`.
+`ArrayController`. Otherwise, you get an `ObjectController`. As of 1.11,
+`ObjectController`s are deprecated and if you try to use its proxy feature you
+will get a warning. Please, take a look at the [deprecations guide] for more
```
Comment (trek) [inline source/concepts/naming-conventions.md]: We've generally been towards inline markdown links. The footnote stye seems to cause regression issues more often when people miss them

File: source/controllers/dependencies-between-controllers.md [hunk 4]
```diff
@@ -42,14 +42,14 @@ its parent `PostController`, which can be done via `controllers.post`
 </ul>
 ```
```
Comment (trek) [inline source/controllers/dependencies-between-controllers.md]: The section above here still has "<h1>Comments for {{controllers.post.title}}</h1>" in it. I think that needs to be changed?

Comment (Serabe) [inline source/controllers/dependencies-between-controllers.md]: Section rephrased.

**Comment by trek** [comment]
Is this ready for review again?

**Comment by Serabe** [comment]
Tomorrow evening, I’ve been sick three days again. These time, amigdalitis :(

-- 
Sergio Arbeo
https://twitter.com/Serabe

On 10 Apr 2015 at 19:46:40, Trek Glowacki (notifications@github.com) wrote:

Is this ready for review again?

**Comment by Serabe** [comment]
I can either fix #90 or this one. I can open a new PR with a fix branch specific for 1.11 (has diverged a lot since this started).

**Comment by trek** [comment]
I'd prefer fixes for master (#90) over older, already published revisions. The plan was never to republish older revisions unless it was absolutely necessary or very, very easy. Priority is always on master.

**Comment by Serabe** [comment]
Shall we close this then?

**Comment by trek** [comment]
If it's a quick fix, let's try. If not, we can just close. Onward and upward!

**Pull request closed without merging**